Lib/asyncio/streams.py

```diff
@@ -753,6 +761,9 @@ def __init__(self, limit=_DEFAULT_LIMIT, loop=None):
             self._loop = events.get_event_loop()
         else:
             self._loop = loop
+            warnings.warn("The loop argument is deprecated since Python 3.8, "
```

Comment:
Private API, people should never create StreamReader class explicitly but use `await open_connection()` and family.

Reply:
removed it